tools/tfm/generate_partition_code.py

```diff
@@ -0,0 +1,689 @@
+#!/usr/bin/python
```

Comment:
This file shares most of the same codes as this one: https://github.com/ARMmbed/mbed-os/blob/master/tools/spm/generate_partition_code.py

There are a decent amount of changes though.

Reply:
Eventually everything under `tools/spm` will be removed when TFM will support dual-core, it will be easier to remove if it does not share stuff with `tools/tfm`.